.github/workflows/stm32-firmware.yml

```
@@ -56,62 +69,63 @@ jobs:
       - name: Run tests
-        working-directory: star-stm32f767-firmware
+        working-directory: star-stm32-firmware
         run: |
-          ctest --test-dir build --output-on-failure --no-tests=error
+          ctest --test-dir ${{ matrix.build_dir }} --output-on-failure --no-tests=error
 
       - name: Build firmware
-        working-directory: star-stm32f767-firmware
-        run: cmake --build build -j$(nproc)
+        working-directory: star-stm32-firmware
+        run: cmake --build ${{ matrix.build_dir }} -j$(nproc)
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2046:warning:1:41: Quote this to prevent word splitting

(shellcheck)